src/verifiers/issuerIdentity/dnsText/openAttestationDnsTxt.v3.test.ts

```diff
@@ -36,18 +36,28 @@ describe("OpenAttestationDnsTxt v3 document", () => {
     const fragment = await openAttestationDnsTxt.verify(documentRopstenValidWithDocumentStore, {
       network: "ropsten",
     });
-    expect(fragment).toStrictEqual({
-      type: "ISSUER_IDENTITY",
-      name: "OpenAttestationDnsTxt",
-      data: { location: "some.io", value: "0x8Fc57204c35fb9317D91285eF52D6b892EC08cD3", status: "INVALID" },
-
-      reason: {
-        code: 1,
-        codeString: "INVALID_IDENTITY",
-        message: "Document issuer identity is invalid",
-      },
-      status: "INVALID",
-    });
+    expect(fragment).toMatchInlineSnapshot(`
+      Object {
+        "data": Object {
+          "location": "some.io",
+          "reason": Object {
```

Comment:
I think I avoided putting this here for v3 previously to avoid duplication of reason (the reason will always be tight to the same issuer for v3, while for v2 it's pushed upfront because it could have been from any of the issuers)

Reply:
Okay it looks silly, removed the nested one already